packages/video_player/video_player/README.md

```diff
@@ -127,3 +127,23 @@ class _VideoAppState extends State<VideoApp> {
   }
 }
 ```
+
+## Usage
+
+The following section contains usage information that goes beyond what is included in the
+documentation in order to give a more elaborate overview of the API.
+
+This is not complete as of now. You can contribute to this section by [opening a pull request](https://github.com/flutter/plugins/pulls).
+
+### Playback speed
+
+You can set the playback speed on your `_controller` (instance of `VideoPlayerController`) by
+calling `_controller.setPlaybackSpeed`. `setPlaybackSpeed` takes a `double` speed value indicating 
+the rate of playback for your video.  
+For example, when given a value of `2.0`, your video will play at 2x the regular playback speed
+and so on.
+
+See the documentation for how different values for the playback speed are handled on the different
```

Comment:
I am not sure if adding additional guidance to the README is a good idea in general. Do you think I should add more information to it (so everything about playback speed there as well) or remove it altogether?

The reason I wanted to add it is because it might be difficult for someone who has not yet used (or installed) the plugin to find out features of the API by navigating through the API reference.

Reply:
I'd just add links to the relevant parts of the documentation, or be more explicit about where to find the info. "For more details, look at file BLAH.dart", or similar. Or link them to the generated API documentation?

Reply:
I think linking to the generated API reference is the way to go - I will update that (but it is not relevant for the part 1/3, so that is fine).